rules.d/70-amd-pmf.rules.in

```diff
@@ -0,0 +1,10 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION!="change", GOTO="pmf_end"
+DRIVERS!="amd-pmf", GOTO="pmf_end"
+
+ENV{EVENT_ID}=="0", RUN+="{{BINDIR}}/systemctl suspend"
```

Comment:
This is just wrong. Please handle these in logind... logind might not be running when udev tries to handle these events, also the actual action to perform should be controllable through logind.conf like power buttons/lid switch.

Reply:
OK.  Let me just do the lock-sessions rule for now then, and the sleep/hibernate will need to get wired up as you suggest.

Reply:
(and there is previous precedent for `lock-sessions` - see *71-seat.rules.in*)